Applications/ctkPluginGenerator/ctkPluginGeneratorMainWindow.ui

```diff
@@ -112,7 +130,16 @@ p, li { white-space: pre-wrap; }
                <property name="spacing">
                 <number>0</number>
                </property>
-               <property name="margin">
+               <property name="leftMargin">
+                <number>0</number>
+               </property>
+               <property name="topMargin">
+                <number>0</number>
+               </property>
+               <property name="rightMargin">
+                <number>0</number>
+               </property>
+               <property name="bottomMargin">
```

Comment:
Are these properties added for convenience because recent version of Qt designer are doing so by default ?

Reply:
Current version of Qt designer added these.